casepro/contacts/models.py

```diff
@@ -241,7 +241,7 @@ def release(self):
         self.is_active = False
         self.save(update_fields=('is_active',))
 
-    def as_json(self, full=False):
+    def as_json(self, full=True):
```

Comment:
Convention for all these `as_json(..)` methods is that by default they return complete representation - and `full=False` returns the shortened representation typically used when object is embedded in another object.